### ISSUE TITLE
Data Catalog Section: Elements must meet minimum color contrast ratio thresholds

### ISSUE THREAD

**Issue by vh8154** [pulibrary/DSS#539]
### Steps to recreate
Go to /catalog
Under data catalog, click the "topic" drop down menu
At the bottom of "topic", select "more »"
Use Axe DevTools on this page

### URL
localhost:3000/catalog

### Issues
color contrast

### Severity
High

### WCAG Success Criterion
[1.4.3 Contrast (minimum)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)

### WCAG Conformance level (A, AA, AAA)
AA

### User impact
Some people cannot read faint text. For all consumers of visual content, adequate light-dark contrast is needed between the relative luminance of text and its background for good readability. 

### Recommended fix
Element has insufficient color contrast of 1.21 (foreground color: #ebe9e2, background color: #ffffff, font size: 10.5pt (14px), font weight: normal). Change to the expected contrast ratio of 4.5:1

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...

### Notes
This also appears for:
Go to /catalog
Under data catalog, click the "country" drop down menu
At the bottom of "country", select "more »"

Related to #526
